woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -149,6 +149,17 @@ echo -n "Making the filesystem usable..." >/dev/console
 # busybox mount /proc before remounting / as rw (full install)
 busybox mount -t proc none /proc 2>/dev/null; STATUS=$((STATUS+$?))
 
+mkdir /sys 2>/dev/null
+busybox mount -t sysfs none /sys ;STATUS=$((STATUS+$?))
+
+#Update microcode if possible
+if [ "$(uname -m | grep -E '^x86_64|^amd64|^i.86')" != "" ] && [ "$(which lscpu)" != "" ]; then
+ #Make sure that cpu is the physical cpu or microcode was not loaded early by boot parameter
+ if [ "$(lscpu | grep "Hypervisor")" == "" ] && [ "$(dmesg | grep "microcode updated early")" == "" ]; then
+  [ "$(which update-microcode)" != "" ] && update-microcode
```

Comment:
Does update-microcode block rc.sysinit? If yes, does ` &` make sense here? Some old and broken Intel CPUs freeze or behave randomly when you hit certain instructions, unless you load the microcode, but I don't know if proceeding to the desktop, etc', can trigger this danger.

Reply:
I think it's better if available microcode was loaded first before continuing init process for security purpose and prevent CPU vulnerability. The update-microcode script that I made has some checks in order to excute microcode loading safely